src/core/layout/gel-button.js

```diff
@@ -6,6 +6,30 @@
 import fp from "../../../lib/lodash/fp/fp.js";
 import * as signal from "../signal-bus.js";
 import * as GameSound from "../game-sound.js";
+import { gmi } from "../gmi/gmi.js";
+
+class Indicator extends Phaser.Sprite {
```

Comment:
Does this need to `extend`? Can we just instantiate a new one inside the `constructor` to make it easier to test?

Reply:
In fact, can we test write tests for this code?